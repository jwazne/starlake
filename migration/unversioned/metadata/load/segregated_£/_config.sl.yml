metadata:
  writeStrategy:
    types:
      OVERWRITE: 'group("mode") == "T" || group("mode") == ""'
      APPEND: 'group("mode") == "D"'
      UPSERT_BY_KEY: 'group("mode") == "A"'
rename: "target_all_inclusive"
database: "{{project_id}}"
