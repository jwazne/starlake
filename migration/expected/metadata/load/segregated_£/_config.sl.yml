---
version: 1
load:
  metadata:
    writeStrategy:
      types:
        OVERWRITE: 'group("mode") == "T" || group("mode") == ""'
        APPEND: 'group("mode") == "D"'
        DELETE_THEN_INSERT: 'group("mode") == "A"'
  rename: "target_all_inclusive"
  database: "{{project_id}}"